tinygrad/runtime/ops_metal.py

```diff
@@ -1,15 +1,58 @@
 from __future__ import annotations
-import os, subprocess, pathlib, ctypes, tempfile, functools
-import Metal, libdispatch
-from typing import List, Any, Tuple, Optional
-from tinygrad.helpers import prod, getenv, DEBUG, unwrap2
-from tinygrad.device import Compiled, Compiler, CompileError, LRUAllocator
+import os, subprocess, pathlib, tempfile, functools
+from typing import List, Any, Tuple, Optional, cast
+from tinygrad.helpers import prod, getenv, DEBUG
+from tinygrad.device import Compiled, Compiler, LRUAllocator
 from tinygrad.renderer.cstyle import MetalRenderer
+import ctypes
+
+class objc_id(ctypes.c_void_p): # This prevents ctypes from converting response to plain int, and dict.fromkeys() can use it to dedup
```

Comment:
These don't have to be released?

Reply:
The intended use case for objc_id is for functions that return something that cannot be released in the runtime, for example objc_getClass returns a class object that's loaded, sel_registerName registers a name in the runtime (repeated call return the same reference); would a different name make it clearer?